proto/prost/private/tests/transitive_dependencies/a_test.rs

```diff
@@ -1,11 +1,11 @@
 //! Tests transitive dependencies.
 
 use a_proto::a::A;
-use b_proto::a::b::B;
-use c_proto::a::b::c::C;
-use duration_proto::google::protobuf::Duration;
-use timestamp_proto::google::protobuf::Timestamp;
-use types_proto::Types;
+use a_proto::b_proto::a::b::B;
```

Comment:
Can you also add a test that shows transitive dependencies become directly consumable as they did before if a `rust_prost_library` is made for it and added as a direct dependency to the the calling crate?

Reply:
I did that in the `transitve_deps_test.rs` which is beside this test.

Reply:
I also updated the original test to take in b_proto as a direct dependency.

Reply:
Thank you! That's exactly what I was thinking